_posts/2020-01-22-cockpit-211.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 211
+author: mvo
+date: 2020-01-22
+tags: cockpit linux
+slug: cockpit-211
+category: release
+summary: Cockpit with certificate and layout improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 211.
+
+### Better support for various TLS certificate formats
+
+Cockpit can now read certificates in a number of formats, such as the
+one from LetsEncrypt.
+
+Specifically, the web server now accepts certificates in
+`/etc/cockpit/ws-certs.d` which have a `.crt` extension, in addition
+to `.cert`. Also, private keys can be in their own file with a `.key`
```

Comment:
I'd say "can now be in a separate file" to emphasize the difference here.

Reply:
Very good!